final/MealsharingNet/IReservationRepository.cs

```diff
@@ -0,0 +1,12 @@
+using MealsharingNet.models;
+
+namespace MealsharingNet;
+
+public interface IReservationRepository
+{
+    Task <IEnumerable<Reservation>> ListReservations();
```

Comment:
You don't have to add a space after Task in these cases `Task <IEnumerable<Reservation>>` => `Task<IEnumerable<Reservation>>`

Reply:
Similar below on line 9 and in the controller